Carthage/Checkouts/Quick

```diff
@@ -1 +1 @@
-Subproject commit 2f037560be197f0f5ae992512549bc29fabb3818
+Subproject commit dc2b4b75d2f718518acbdef289a72fad77975a8a
```

Comment:
The submodule commit needs to be reverted as well.

(Please post a comment when you're done so I get a notification!)